config.json

```diff
@@ -1,8 +1,8 @@
 {
-    "deep_filter_path": "MediaProcessor/res/deep-filter-0.5.6-x86_64-unknown-linux-musl",
-    "deep_filter_tarball_path": "MediaProcessor/res/DeepFilterNet3_ll_onnx.tar.gz",
-    "deep_filter_encoder_path": "MediaProcessor/res/DeepFilterNet3_ll_onnx/tmp/export/enc.onnx",
-    "deep_filter_decoder_path": "MediaProcessor/res/DeepFilterNet3_ll_onnx/tmp/export/df_dec.onnx",
+    "deep_filter_path": "MediaProcessor\\res\\deep-filter-0.5.6-x86_64-unknown-linux-musl",
+    "deep_filter_tarball_path": "MediaProcessor\\res\\DeepFilterNet3_ll_onnx.tar.gz",
+    "deep_filter_encoder_path": "MediaProcessor\\res\\DeepFilterNet3_ll_onnx\\tmp\\export\\enc.onnx",
+    "deep_filter_decoder_path": "MediaProcessor\\res\\DeepFilterNet3_ll_onnx\\tmp\\export\\df_dec.onnx",
```

Comment:
This may fix it for windows, but breaks it for Unix. While testing this PR on macOS I got some panics from the library which was caused by a file not found due to this. This happened after running the launcher. 

Please dynamically update the paths for Windows and let's keep the default paths with the Unix slashes. There seems to be an issue with the dynamic update of this. Could you please check ? Thanks

Reply:
The issue is launcher.py successfully updates the path but config.json modified can be accidently pushed with other changes (In this case).

we can change them back after running but what if user runs via command python app.py. That will not work because of paths. So for windows development we have to be careful not to push the config.json.

Reply:
Well, the launcher will be the official method to launch any application or even for setting up the repository. Of course it's open source, so everyone is free to hack-away at the code, but we don't necessarily have to support use cases that fall outside of the officially supported methods -that's why we provide a launcher. 

I agree with you on your other point though. I'd suggest exploring a runtime override (potentially even a second `runtime_config.json`) instead of directly overwriting the file. The config.json would serve as a base. 

On a second note, we should add a check on app.py to warn the user if it's called from outside the launcher.

Reply:
we should add a runtime json that we dont have to warn user about launching outside the launcher.py
we should standardize to use launcher.py